WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -35,7 +35,7 @@ class WordPressAuthenticationManager: NSObject {
                                                                 googleLoginServerClientId: ApiCredentials.googleLoginServerClientId(),
                                                                 googleLoginScheme: ApiCredentials.googleLoginSchemeId(),
                                                                 userAgent: WPUserAgent.wordPress(),
-                                                                showLoginOptions: true,
+                                                                showLoginOptions: !FeatureFlag.unifiedWordPress.enabled,
```

Comment:
I don't think this should change; it should always be `true` as it was before.

Unless I'm missing something, this only controls the ability to login via Google on the old Email Address view. WPiOS didn't show this before, and shouldn't ever. (I believe this flag is specifically for Woo.)